general/server/media/music.md

```diff
@@ -54,6 +54,8 @@ Albums with several discs will be fine with the metadata tags, but you can also
 
 Images will be scraped from album or artist folders, and they can also be embedded in the music files themselves. The supported filenames are listed below for each respective image.
 
+[!Note] You need to enable those two plugins AudioDB and MusicBrainz to get poster image from artists.
```

Comment:
I don't think this is the right format for pretty note entries in the documentation.

Reply:
fixed